README.md

```diff
@@ -9,14 +9,53 @@ which incidents. Its deployment happens once a day via
 
 ## Getting Started
 
-  To get started all you need is an empty PostgreSQL database and the following dependencies:
+### Install Dependencies
 
-    $ sudo zypper in -C postgresql-server postgresql-contrib
-    $ sudo zypper in -C perl-Mojolicious perl-Mojolicious-Plugin-Webpack \
-      perl-Mojo-Pg perl-Cpanel-JSON-XS perl-JSON-Validator perl-IO-Socket-SSL nodejs16
-    $ npm install
+To install all required dependencies, run
 
-Update the config file `dashboard.yml` to point to your PostgreSQL database (and other services where appropriate):
+    sudo zypper in -C postgresql-server postgresql-contrib
+    sudo zypper in -C perl-Mojolicious perl-Mojolicious-Plugin-Webpack \
+      perl-Mojo-Pg perl-Cpanel-JSON-XS perl-JSON-Validator perl-IO-Socket-SSL nodejs18
+    npm install
+
+if you are on an apt-based system, run 
+
+    npx playwright install --with-deps
+
+if you are on a zypper-based system, omit the `--with-deps` parameter
+
+    npx playwright install
+
+and ignore the missing dependencies
+
+#### nodejs18
+
+In case `nodejs18` is not available (e.g. Tumbleweed), you can alternatively get it via the [Node Version Manager](https://github.com/nvm-sh/nvm). Simply [install nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#install--update-script) then run `nvm install 18` and `nvm use 18`.
```

Comment:
What is this needed for? `npm install` worked fine for me on Tumbelweed.

Reply:
Good point. `npm install` works also just fine for me. There is no need to install version 18 specifically.

Reply:
Should be 18 or newer.

Reply:
currently the README says one should install nodejs16, which is not available on TW. As production was using nodejs18 when i first edited the docs, i added the section on how to install that version on TW using nvm. I have now removed that paragraph and added nodejs22 (also available on TW), to the zypper install line in the setup instructions.